### DISCUSSION TITLE
[월드 1.1.7] OnDrawGizmos

### DISCUSSION THREAD

**Discussion by smilejsu82** [naverz/zepeto-studio-kor#614] [5. Q&A]
테스트를 위해 OnDrawGizmos를 사용하고 싶은데요 
현재는 수동으로 C# 스크립트를 만들어 넣어 주면서 테스트 중입니다 
물론 빌드 할때는 제거 후 빌드 할 예정이구요 

OnDrawGizmos 지금 타입스크립트에서 가능한가요?

C# 스크립트 테스트용을 만든것들 타입스크립트에서 동적으로 AddComponent할수 있는 방법은 없을까요?
특히 ZepetoPlayer.character에 붙이고 싶습니다 : )
도와주세요 ~ !!!! 

![image](https://user-images.githubusercontent.com/50188795/163110201-572c45e2-e7bf-4175-b65c-951a8dd84462.png)

https://user-images.githubusercontent.com/50188795/163110207-fc7b28b2-4cf0-4998-b8aa-3eaf513316a4.mp4

**Comment by Cha-SeongHun** [comment]
C#스크립트가 붙어있는 Prefab을 원하는 오브젝트의 자식으로 추가하는 방법으로 우회해서 사용 가능해 보입니다!

경험상 OnDrawGizmos 함수는 Typescript상에서 동작이 되지 않았고 저도 그냥 C#스크립트로 작성해서 사용했었어요

**Reply by smilejsu82** [reply]
좋은 방법이네요 후... 
웃는게 웃는게 아니네요 😂

**Reply by smilejsu82** [reply]
감사합니다 덕분에 해결 되었습니다 ^^

https://user-images.githubusercontent.com/50188795/163114245-bb221db3-e5d6-422c-914e-1f6bbd1e3ca5.mp4
